scripts/lib/custom_networking.sh

```diff
@@ -0,0 +1,65 @@
+func configure_custom_networking(){
```

Comment:
nit: consider adding description at the top on what this function does

Reply:
Yes comments are pending with this PR, I will add it before marking it to ready for review.